components/pages/about/CovidCloudPane.tsx

```diff
@@ -46,20 +46,20 @@ const CovidCloudPane = (): ReactElement => {
         width: calc(100% - 100px);
       `}
     >
-      <img src="/images/covid-cloud.png" alt="logo for Covid Cloud" width="180" />
+      <img src="/images/viral-ai.png" alt="logo for Viral AI" width="180" />
```

Comment:
I see a reference here to a file that doesn't currently exist in the repo, and is not being added with the PR.
Going to assume it's a mistake?

Reply:
Hmm weird I think the file was added in my previous commits/pushes as we discussed yesterday?  I thought the file would get pushed with my branch

Reply:
File uploaded to this branch, please re-check!